src/utils/__tests__/get-code-from-ast.spec.ts

```diff
@@ -22,7 +22,7 @@ import a from 'a';
         importOrderGroupNamespaceSpecifiers: false,
         importOrderSortSpecifiers: false,
     });
-    const formatted = getCodeFromAst(sortedNodes, code, null);
```

Comment:
Is it possible to have one more test case with directives?

Reply:
Done.